### DISCUSSION TITLE
how to add new localization file?

### DISCUSSION THREAD

**Discussion by zprofreelancer** [serenity-is/Serenity#5379] [Help]
Hello,
I'm trying to add new language "arabic"...so i added new language from administration link then add the file site.texts.ar-EG.json in "scripts/site/texts". the website direction is right to left but the content still in english. so, what is the problem?

Thanks.

**Comment by afetouh** [comment]
Hello,
How you added exactly the Arabic, because I have added it with language id = ar and second time with ar-EG, but the interface does not change the direction to RTL

**Reply by zprofreelancer** [reply]
I have added the language id = ar-EG then added the file in site.texts.ar-EG.json in "~/scripts/site/texts".  you can download the localization file from here: https://github.com/serenity-is/Serenity/issues/3574. and change the file name from user.texts.ar-EG.json to site.texts.ar-EG.json

**Reply by zprofreelancer** [reply]
Problem solved. i removed the comments on the file user.texts.ar-EG.json

**Comment by samaremad** [comment]
hello did you make your project in arabic and it work correctly without problems ?
